int64 Procedure
baseStruct[] structList
string FilePath